test/addons/rook-cephfs/test

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+
+from drenv import kubectl
+
+PVC_NAME = "cephfs-pvc"
+SNAP_NAME = "cephfs-snap"
+
+
+def test_provisioning(cluster):
+    print(f"Deploying pvc {PVC_NAME} in cluster '{cluster}'")
+    kubectl.apply(
+        f"--filename={PVC_NAME}.yaml",
+        "--namespace=rook-ceph",
+        context=cluster,
+    )
+
+    print(f"Waiting until pvc {PVC_NAME} is bound in cluster '{cluster}'")
+    kubectl.wait(
+        f"pvc/{PVC_NAME}",
+        "--for=jsonpath={.status.phase}=Bound",
+        "--namespace=rook-ceph",
+        "--timeout=300s",
+        context=cluster,
+    )
```

Comment:
Would be useful to use the bounded PVC - for example start a busybox pod with it, so we know that we can read and write to it.

Reply:
Why? Firstly these tests are not for every component, we do not do the same with RBD volumes, it is a sanity for the component that is added. I am unsure how much testing we want to do here and for what reason.

Reply:
The fact we don't test that rbd pvc works is also bad, but so far it worked for us. We can use the volsync test for validation (after switching to cephfs instead of csi-hostpath-driver).